code/modules/vore/fluffstuff/custom_items_vr.dm

```diff
@@ -558,4 +558,11 @@ obj/item/weapon/material/hatchet/tacknife/combatknife/fluff/katarina/handle_shie
 		assignment = user.job
 		user.set_id_info(src)
 		configured = 1
-		user << "<span class='notice'>Card settings set.</span>"
\ No newline at end of file
+		user << "<span class='notice'>Card settings set.</span>"
+
+//WickedTempest: Chakat Tempest
+/obj/item/weapon/reagent_containers/hypospray/vr/tempest
+	desc = "A custom-made MKII hypospray belonging to Chakat Tempest. There's small print engraved on the handle: A medicine-cat has no time for doubt. Act now, act swiftly."
+	icon = 'icons/obj/syringe_vr.dmi'
```

Comment:
Belongs in custom_items_vr.dmi not syringe_vr.dmi, like the other fluff hypospray.

On your branch, reset mixed to the previous commit, copy your sprite to custom_items_vr.dmi, discoard your syringe_vr.dmi change, and update the custom_items_vr.dm file to point to the right .dmi file.